layouts/download/download.html

```diff
@@ -29,6 +29,7 @@ <h3>Binary Distribution</h3>
 
             {{ $major := "" }}
             {{ $cnt := 1 }}
+            <div class="table-wrapper">
```

Comment:
Can be removed.

Reply:
This is required for the table within the downloads page. Without this, the table will make the page unresponsive in the mobile version.